pkg/operation/botanist/extension.go

```diff
@@ -104,6 +104,10 @@ func mergeExtensions(registrations []gardencorev1beta1.ControllerRegistration, e
 			}
 
 			if res.GloballyEnabled != nil && *res.GloballyEnabled {
+				// For workerlessShoots, check if the extension supports it
```

Comment:
This doc string is useless since the next line is already quite readable and tells the same. Remove it.